### ISSUE TITLE
NTR: Absent sebaceous glands

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4731]
**Preferred term label:**
Absent sebaceous glands

**Synonyms**
absence of sebaceous glands

**Definition (free text, please give PubMed ID)**
Absence of the sebaceous gland, the holocrine gland that secretes sebum into the hair follicles, or in hairless areas into ducts.
dbxref: MP:0000647

'has part' some 
    (absent
     and ('inheres in' some 'sebaceous gland')
     and ('has modifier' some abnormal))

**Parent term (use hpo.jax.org/app)**
HP_0011138 'Abnormality of skin adnexa morphology'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601553
For phenopacket case report: Blanco-Kelly-2017-ABCA4

See TheJacksonLaboratory/LIRICAL#180

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by pnrobinson** [comment]
@nicolevasilevsky Looks good but the parent should be https://hpo.jax.org/app/browse/term/HP:0032226

**Comment by nicolevasilevsky** [comment]
Thanks, I can add this.

Re: the parent - that crossed my mind, but is an absent anatomical an abnormal morphology? If it is absent, it can't have a morphology, right?

**Comment by nicolevasilevsky** [comment]
I see other absent classes are classified as children of 'abnormal morphology' though - I will add this as a subclass of 'Abnormal sebaceous gland morphology'

thanks, @pnrobinson!

**Comment by pnrobinson** [comment]
There has been a lot of debate about where to put the absent, but I think the best compromise is to put Absent X under Abnormal X morphology -- this is what the HPO has now for almost all cases, and if there is something else we should fix it!
